NumRush.py

```diff
@@ -351,6 +359,14 @@ def run(self):
             pygame.display.update()
             fpsClock.tick(FPS)
 
+            if score_flag:
+                sleep(0.5)
```

Comment:
Could you try and use `pygame.time.delay` or `pygame.time.wait` and see how that compares to this?

Reply:
It works fine with ` pygame.time.delay` too. Is it recommended to use it instead?

Reply:
Yes I'd recommend you use that instead.

Reply:
right, I'll update.